scripts/compute_mappings/compute_ancestor_mapping.py

```diff
@@ -0,0 +1,174 @@
+#!/usr/bin/env python
```

Comment:
Add somewhere what these were converts from and using what for reproducibility.

Reply:
Added a README to the directory